src/common/platform.ts

```diff
@@ -1,3 +1,4 @@
 export const isWindows = process.platform === 'win32';
 export const isMacintosh = process.platform === 'darwin';
 export const isLinux = process.platform === 'linux';
+export const isFlatpak = !!process.env['FLATPAK_ID'];
```

Comment:
There seems to be other envvar that could be used, let me know if you would like me to consolidate that